listenbrainz/db/model/user_timeline_event.py

```diff
@@ -27,7 +27,7 @@
 from listenbrainz.db.msid_mbid_mapping import MsidMbidModel
 
 
-class UserTimelineEventType(Enum):
+class UserTimelineEventType(str, Enum):
```

Comment:
Since we are on Python 3.11 in LB, you can use StrEnum here. https://docs.python.org/3/library/enum.html#enum.StrEnum